File: java-micronaut-jackson/src/main/java/test_service/services/CheckServiceImpl.java
package test_service.services;

import io.micronaut.context.annotation.Bean;
import test_service.models.*;
import test_service.services.check.*;

@Bean
public class CheckServiceImpl implements CheckService {
	@Override
	public void checkEmpty() {
		return;
	}
	@Override
	public void checkEmptyResponse(Message body) {
		return;
	}

	@Override
	public CheckForbiddenResponse checkForbidden() {
		return new CheckForbiddenResponse.Forbidden();
	}

	@Override
	public SameOperationNameResponse sameOperationName() {
		return new SameOperationNameResponse.Ok();
	}
}

File: kotlin-micronaut-jackson/src/main/kotlin/test_service/services/CheckServiceImpl.kt
package test_service.services

import io.micronaut.context.annotation.Bean
import test_service.models.*
import test_service.services.check.*

@Bean
class CheckServiceImpl : CheckService {
    override fun checkEmpty() {
        return
    }

    override fun checkEmptyResponse(body: Message) {
        return
    }

    override fun checkForbidden(): CheckForbiddenResponse {
        return CheckForbiddenResponse.Forbidden()
    }

    override fun sameOperationName(): SameOperationNameResponse {
        return SameOperationNameResponse.Ok()
    }
}
